### PR TITLE
Performance Improvements for uwebsocket.js

### PR DESCRIPTION
**Pull request by srisaiswaroop**
1. Added Slow-JSON-Stringify based JSON serialization to improve the speed.

2. Used a different sort method which is sligthly faster and gives better performance for this usecase.

3. Updated to latest postgres module

4. Made random number generator slightly better by avoiding the addition.

### CODE REVIEW

**Review by srisaiswaroopwenergy — APPROVED** [review]

File: frameworks/JavaScript/uwebsockets.js/src/utils.js [hunk 1]
```diff
@@ -57,3 +59,29 @@ export function escape(text) {
   if (unsafeHTMLMatcher.test(text) === false) return text;
   return text.replace(unsafeHTMLMatcher, function (m) { return escapeHTMLRules[m] || m; });
 }
+
+/**
+ * Using Slow json stringify module to get faster results as explained in just-js blog post
```
Comment (masfahru) [inline frameworks/JavaScript/uwebsockets.js/src/utils.js:64]: Could you please remove the phrase "as explained in just-js blog post" and add the link to the blog post in the PR comment?

Comment (srisaiswaroopwenergy) [reply frameworks/JavaScript/uwebsockets.js/src/utils.js:64]: Sure will add it. Meanwhile this is the post i was referring to. 
https://just.billywhizz.io/blog/on-javascript-performance-01/

File: frameworks/JavaScript/uwebsockets.js/src/utils.js [hunk 2]
```diff
@@ -57,3 +59,33 @@ export function escape(text) {
   if (unsafeHTMLMatcher.test(text) === false) return text;
   return text.replace(unsafeHTMLMatcher, function (m) { return escapeHTMLRules[m] || m; });
 }
+
+/**
+ * Using Slow json stringify module to get faster results as explained in just-js blog post below
```
Comment (masfahru) [inline frameworks/JavaScript/uwebsockets.js/src/utils.js:64]: Thank you, but sorry if I wasn't clear enough. What I mean is, could you remove any references in the code comments that are not directly related to uWebsockets.js? Instead, you can put the references in the PR conversation section.

**Review by masfahru — APPROVED** [review]

**Comment by srisaiswaroopwenergy** [comment]
@nbrady-techempower Can you approve the "1 workflow awaiting approval". 
Also this is ready for merge.
